data/transactions/logic/opcodes.go

```diff
@@ -294,6 +294,9 @@ var OpSpecs = []OpSpec{
 	{0x94, "exp", opExp, asmDefault, disDefault, twoInts, oneInt, 4, modeAny, opDefault},
 	{0x95, "expw", opExpw, asmDefault, disDefault, twoInts, twoInts, 4, modeAny, costly(10)},
 
+	//More Hash Functions
+	{0x96, "sha3_256", opSHA3_256, asmDefault, disDefault, oneBytes, oneBytes, 6, modeAny, costly(130)},
```

Comment:
How did you derive the cost?

Reply:
I wasn't sure how to derive the cost. The cost of the SHA3-256 should be roughly the same as the Keccak256 as they are very similar algorithms, except the padding in the SHA3-256 adds up to 4 additional bits. Im open to any suggestions of how to derive this cost more accurately?

Reply:
There is some benchmarking code in eval_test.go.  But hold off for a couple days. We're in the middle of a merge that will change it slightly.  But once that merge happens, you'll find
`func BenchmarkBase64Decode(b *testing.B)` that was used to compare base64 decode to the costs of various hashes.  By adding sha3_256 in there, we should be able to get a feel for relative costs.

(The function may already be in master, if you want to look now.)

Reply:
Thanks for the information! Which merge should I be looking out for?

Reply:
feature/contract-to-contract going into master

Reply:
Thanks for the heads up @algoanne! Ill get on this as soon as possible

Reply:
To me they are in the same ball park as one another and thus should have the same cost, as the eqv for other hashes like
BenchmarkBase64Decode/sha256_small-8             	 1260788	      1003 ns/op
BenchmarkBase64Decode/sha512_256_small-8         	 1695235	       715.2 ns/op

Are very different

Reply:
Yes, it sounds like it should have the same cost as keccak.  I think we can take this as soon as CI passes.  I'll be making a pass over it once that happens as I have changed the opcode interface a bit.  But I think your work is done! Thanks for the contribution.